t/discovery/consul_kv.t

```diff
@@ -218,6 +218,7 @@ routes:
 GET /hello
 --- response_body_like eval
 qr/server [1-2]/
+--- ignore_error_log
```

Comment:
I check all `--- ignore_error_log`, and I confirm they are necessary because retries of these test cases occurrences are random, so you could not assert they happen or not. After all, they are just retries, and do not affect the test goals, so we have to ignore them.

Reply:
And some are health checker retries.